.editorconfig

```diff
@@ -0,0 +1,11 @@
+# https://editorconfig.org
+root = true
+
+[*.{cpp,h}]
+indent_style = space
+indent_size = 4
+end_of_line = lf
```

Comment:
Let's remove this line because I'm not sure `lf` is used on Windows, it probably depends on your git settings.